tests/integration/defs/accuracy/test_llm_api_pytorch.py

```
@@ -2950,6 +2950,111 @@ def test_nvfp4_2_model_mtp(self, tp_size, cuda_graph, overlap_scheduler,
+    def test_nvfp4_2_model_mtp(self, tp_size, cuda_graph, overlap_scheduler,
+                               chunked_prefill, max_batch_size, moe_backend):
+
+        kv_cache_config = KvCacheConfig(free_gpu_memory_fraction=0.70)
+        pytorch_config = dict(
+            disable_overlap_scheduler=not overlap_scheduler,
+            cuda_graph_config=CudaGraphConfig() if cuda_graph else None,
+            moe_config=MoeConfig(backend=moe_backend))
+
+        mtp_config = MTPDecodingConfig(num_nextn_predict_layers=3,
+                                       mtp_eagle_one_model=False,
+                                       speculative_model_dir=model_path)
```

Ruff (0.14.11)

Undefined name `model_path`

(F821)